project-base/tests/FrontendApiBundle/Functional/Payment/PaymentTest.php

```diff
@@ -0,0 +1,58 @@
+<?php
+
+declare(strict_types=1);
+
+namespace Tests\FrontendApiBundle\Functional\Payment;
+
+use Shopsys\FrameworkBundle\Component\Domain\Domain;
+use Tests\FrontendApiBundle\Test\GraphQlTestCase;
+
+class PaymentTest extends GraphQlTestCase
+{
+    /**
+     * @var \Shopsys\FrameworkBundle\Model\Payment\PaymentFacade
+     * @inject
+     */
+    protected $paymentFacade;
+
+    /**
+     * @var \Shopsys\FrameworkBundle\Model\Payment\Payment
+     */
+    protected $payment;
+
+    protected function setUp(): void
+    {
+        $this->payment = $this->paymentFacade->getById(2);
+
+        parent::setUp();
+    }
+
+    public function testCategoryNameByUuid(): void
```

Comment:
I don't see a context in the method name :thinking:

Reply:
forgotten copy paste, fixed